test/integration/windows/ops-agent/2.0.1/default_test.rb

```diff
@@ -31,7 +31,5 @@
   if node['package_state'] == 'present'
     it { should exist }
     its('sha256sum') { should eq 'a8dd6a4312fb2d62aa11bccd612182033fd1b378b9bc6da8eef25cee54b15dd3' }
-  else
-    it { should_not exist }
```

Comment:
Is the `test/integration/windows/ops-agent/2.0.1/default_test.rb` file still needed or can we delete it now?

Reply:
Deleted 👍